src/components/artwork_filter/headline.tsx

```diff
@@ -77,4 +84,14 @@ const StyledHeadline = styled(Headline)`
   font-size: 2em;
 `
 
-export default StyledHeadline
+export default Relay.createContainer(StyledHeadline, {
+  fragments: {
+    facet: () => Relay.QL`
+      fragment on ArtworkFilterFacet {
+        ... on ArtworkFilterTag {
```

Comment:
Here's where we get the name of the facet we are filtering on. Currently MP only supports the Tag facet, but it could support `Gene`, `Artist`, `Sale`, etc.